src/sections/channels.tsx

```diff
@@ -0,0 +1,492 @@
+    navigate(`#${newHash}`, { replace: true });
+  };
+
+  useEffect(() => {
+    handleRefresh();
+    navigate(`#incoming`, { replace: true });
+  }, [loginData]);
+
+  const handleRefresh = () => {
+    dispatch(
+      actionsAsync.getChannelsThunk({
+        apiEndpoint: loginData.apiEndpoint!,
+        apiToken: loginData.apiToken!,
+      })
+    );
+    dispatch(
+      actionsAsync.getAliasesThunk({
+        apiEndpoint: loginData.apiEndpoint!,
+        apiToken: loginData.apiToken!,
+      })
+    );
+  };
+
+  const getAliasByPeerId = (peerId: string): string => {
+    if (aliases) {
+      for (const [alias, id] of Object.entries(aliases)) {
+        if (id === peerId) {
+          return alias;
+        }
+      }
+    }
+    return peerId; // Return the peerId if alias not found for the given peerId
+  };
+
+  const exportToCsvButton = () => {
+    const tabLabel = tabIndex === 0 ? 'incoming' : 'outgoing';
+    const channelsData =
+      tabIndex === 0 ? channels?.incoming : channels?.outgoing;
+
+    return (
+      <button
+        disabled={!channelsData || Object.keys(channelsData).length === 0}
+        onClick={() => {
+          if (channelsData) {
+            exportToCsv(
+              Object.entries(channelsData).map(([, channel]) => ({
+                channelId: channel.channelId,
+                peerId: channel.peerId,
+                status: channel.status,
+                dedicatedFunds: channel.balance,
+              })),
+              `${tabLabel}-channels.csv`
+            );
+          }
+        }}
+      >
+        export
+      </button>
+    );
+  };
+
+  const handleCloseChannels = (
+    direction: 'incoming' | 'outgoing',
+    peerId: string,
+    channelId: string
+  ) => {
+    set_closingStates((prevStates) => ({
+      ...prevStates,
+      [channelId]: {
+        closing: true,
+        closeSuccess: false,
+        closeErrors: [],
+      },
+    }));
+
+    dispatch(
+      actionsAsync.closeChannelThunk({
+        apiEndpoint: loginData.apiEndpoint!,
+        apiToken: loginData.apiToken!,
+        direction: direction,
+        peerId: peerId,
+      })
+    )
+      .unwrap()
+      .then(() => {
+        set_closingStates((prevStates) => ({
+          ...prevStates,
+          [channelId]: {
+            closing: false,
+            closeSuccess: true,
+            closeErrors: [],
+          },
+        }));
+        handleRefresh();
+      })
+      .catch((e) => {
+        set_closingStates((prevStates) => ({
+          ...prevStates,
+          [channelId]: {
+            closing: false,
+            closeSuccess: false,
+            closeErrors: [
+              ...(prevStates[channelId]?.closeErrors || []),
+              {
+                error: e.error,
+                status: e.status,
+              },
+            ],
+          },
+        }));
+      });
+  };
+
+  const fundChannel = (channelId: string, peerId: string) => {
```

Comment:
if its a React Component, it should start with a big letter, and also it should be more clear what it does like: `FundChannelModal`.
Put it into the components folder, as we will you it in more sections